### DISCUSSION TITLE
Community Partner for Global Hackathon " Hack The Mountains 5.0 "

### DISCUSSION THREAD

**Discussion by Anmolreshi** [pydelhi/conference#214] [Community Partnership]
**About Hack The Mountains:**

Hack The Mountains is a globally acclaimed hackathon that has become a beacon for innovation, creativity, and collaboration. Over the past four seasons, we have brought together **12,000+** talented developers, designers, and hackers from around the world to create groundbreaking solutions and address real-world challenges. This year, we are taking it a step further by hosting a **30-hour hybrid hackathon**, allowing participants to join us both online and offline. This hybrid format not only broadens participation but also infuses a fresh dynamism into the event.

Having successfully hosted two virtual events and two hybrid events, we are excited to announce that HTM 5.0 will also adopt a hybrid format, featuring both online and offline participation.

**Event Details:**

**Date:** September 14-15, 2024

**Mode:** Hybrid Hackathon (Online + Offline)

**Venue:** Gujarat , India

**Website**: [[Hack The Mountains]](https://hackthemountain.tech/)

Here are some potential areas for collaboration:

1. Brand Visibility: Logo on website and all over relevant social media posts.

2. Community Promotion: Promote Hack The Mountains on your social groups and platforms, connections and Websites.

3. Active Participation: Encourage your community members to join HTM 5.0 events and you can enjoy exclusive swag, goodies, and mementos.

4. Exclusive Offline Access: A unique chance for in-person networking and collaboration during the hackathon.

This partnership will not only provide immense value to your community members but also help in showcasing the innovative spirit of your community on a larger platform.

**Comment by Daksh777** [comment]
Hey Anmol

We're good to go with this partnership!
